composer.json

```diff
@@ -7,7 +7,7 @@
         "ext-json": "*",
         "jetbrains/phpstorm-stubs": "2019.3",
         "nikic/php-parser": "^4.3.0",
-        "phpdocumentor/reflection-docblock": "^4.3.4",
+        "phpdocumentor/reflection-docblock": "^5.0.0",
```

Comment:
@Ocramius since there are no BC breaks that seemingly affect us (I have not checked personally but the build will tell all, I'm sure...) should we support `^4.3.4|^5.0` do you think?

Reply:
Nay, let's bump and ship a new minor, please. Supporting both is not worth it (from a potential bugs perspective)

Reply:
Aight 👍